components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ApiResource.java

```diff
@@ -45,42 +42,18 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-import javax.xml.stream.XMLStreamException;
-
-import static org.wso2.carbon.micro.integrator.core.internal.ServiceComponent.getServerConfigurationService;
-import static org.wso2.carbon.micro.integrator.management.apis.Utils.getQueryParameters;
 
 public class ApiResource extends APIResource {
 
-    private static Log log = LogFactory.getLog(ApiResource.class);
-
-    private static final String ROOT_ELEMENT_APIS = "<APIs></APIs>";
-    private static final String COUNT_ELEMENT = "<Count></Count>";
-    private static final String LIST_ELEMENT = "<List></List>";
-    private static final String LIST_ITEM = "<Item></Item>";
-
-    private static final String ROOT_ELEMENT_API = "<API></API>";
-    private static final String NAME_ELEMENT = "<Name></Name>";
-    private static final String CONTEXT_ELEMENT = "<Context></Context>";
-    private static final String URL_ELEMENT = "<Url></Url>";
-    private static final String HOST_ELEMENT = "<Host></Host>";
-    private static final String PORT_ELEMENT = "<Port></Port>";
-    private static final String VERSION_ELEMENT = "<Version></Version>";
-
-    private static final String RESOURCES_ELEMENT = "<Resources></Resources>";
-    private static final String RESOURCE_ELEMENT = "<Resource></Resource>";
-    private static final String METHOD_ELEMENT = "<Methods></Methods>";
-    private static final String STAT_ELEMENT = "<Stats></Stats>";
-    private static final String TRACING_ELEMENT = "<Tracing></Tracing>";
-
+    private Utils utils = new Utils();
```

Comment:
What I meant is not to remove static methods. :. Let's keep the static methods but not use static imports. Instead let's call them explicitly. E.g. `Util.getQueryParameters(axis2MessageContext);`